### PR TITLE
freecad: fix build

### PR DESCRIPTION
**Pull request by catap**
#### Description

Unfortunately update to the last version requires to recreate from scratch some used patches.

###### Type(s)

- [x] bugfix

###### Tested on

macOS 12.6 21G115 x86_64
Xcode 14.1 14B47b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-pivy/Portfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ description         Pivy is a Coin binding for Python.
 long_description    ${description}
 platforms           darwin
 
-python.versions     27 35 36
+python.versions     27 35 36 37 38 39 310
```
Comment (reneeotten) [inline python/py-pivy/Portfile:17]: are the old Python subports still used anywhere as dependents or could they be removed?

**Comment by reneeotten** [comment]
ping @mbrethen - any comments on this PR before we merge?

File: python/py-pivy/Portfile (hunk 1 above)
Comment (catap) [reply python/py-pivy/Portfile:17]: @reneeotten `py27` is used by freecad ;)

Comment (catap) [reply python/py-pivy/Portfile:17]: @reneeotten it is used only by freecad. I may remove old versions without any problems.

Comment (catap) [reply python/py-pivy/Portfile:17]: done

**Comment by reneeotten** [comment]
@catap CI failure die to checksum mismatch

**Comment by catap** [comment]
@reneeotten that's strange because only one from two CI had failed

**Comment by catap** [comment]
Enforced `github.tarball_from releases` to match checksum and reuse `pivy-0.6.2.tar.gz` which is already on distfiles.

**Pull request merged**